Azure/sonic-buildimage
# Convert syncd-vs docker to buster

Convert syncd-vs docker to buster

**- Why I did it**
This is an enhancement.

**- How I did it**
Code change.

**- How to verify it**
Build syncd-vs docker.
Check that "/etc/apt/sources.list" in syncd docker references buster.

**- Description for the changelog**
Changes to convert syncd-vs docker to buster.

Signed-off-by: Joyas Joseph <joyas_joseph@dell.com>

**- A picture of a cute animal (not mandatory but encouraged)**
